src/docfx/lib/markdown/MarkdigUtility.cs

```diff
@@ -13,6 +17,34 @@ namespace Microsoft.Docs.Build
 {
     internal static class MarkdigUtility
     {
+        private static readonly IReadOnlyDictionary<Type, string> s_markdownElementTypeMapping = new Dictionary<Type, string>()
```

Comment:
I tent to like putting this into `MarkdownElementTelemetryExtension`, the `MarkdigUtility` was intended for common shared functions, this function is telemetry only, so